src/views/UrbitOS/Home.js

```diff
@@ -0,0 +1,188 @@
+import React, { useCallback, useState } from 'react';
+import { Grid, Button, SelectInput } from 'indigo-react';
+import * as ob from 'urbit-ob';
+import { azimuth } from 'azimuth-js';
+
+import { usePointCursor } from 'store/pointCursor';
+import { usePointCache } from 'store/pointCache';
+
+import useCurrentPointName from 'lib/useCurrentPointName';
+import useKeyfileGenerator from 'lib/useKeyfileGenerator';
+import * as need from 'lib/need';
+
+import {
+  OutButton,
+  ForwardButton,
+  BootUrbitOSButton,
+} from 'components/Buttons';
+import CopyButton from 'components/CopyButton';
+import NetworkingKeys from 'components/NetworkingKeys';
+
+import BridgeForm from 'form/BridgeForm';
+import { useLocalRouter } from 'lib/LocalRouter';
+
+export default function UrbitOSHome() {
+  const { pointCursor } = usePointCursor();
+  const { getDetails } = usePointCache();
+
+  const { push, names } = useLocalRouter();
+
+  const point = need.point(pointCursor);
+  const details = need.details(getDetails(point));
+
+  const sponsor = ob.patp(details.sponsor);
+
+  const { code, notice } = useKeyfileGenerator();
+
+  const [showKeys, setShowKeys] = useState(false);
+
+  const showSponsor = azimuth.getPointSize(point) !== azimuth.PointSize.Galaxy;
+  const toggleShowKeys = useCallback(() => setShowKeys(!showKeys), [
+    setShowKeys,
+    showKeys,
+  ]);
+
+  const goNetworkingKeys = useCallback(() => push(names.NETWORKING_KEYS), [
+    names,
+    push,
+  ]);
+
+  const goChangeSponsor = useCallback(() => push(names.CHANGE_SPONSOR), [
+    push,
+    names,
+  ]);
+  return (
+    <>
+      <Grid>
+        <Grid.Item full className="mv7">
+          Urbit OS
+        </Grid.Item>
+        <Grid.Divider />
+        <Grid.Item full as={BootUrbitOSButton} />
+      </Grid>
+
+      <Grid>
+        <Grid.Item full className="mv7 f5">
+          Network
+        </Grid.Item>
+        <Grid.Divider />
+        {showSponsor && (
+          <>
+            <Grid.Item
+              full
+              as={ForwardButton}
+              detail="A sponsor finds new peers in your network"
+              accessory={<u>Change</u>}
+              onClick={goChangeSponsor}>
+              <span className="mono">{sponsor}</span>
+              <span className="f7 bg-black white p1 ml2 r4">SPONSOR</span>
```

Comment:
Why is this here? This kind of element isn't used anywhere else yet, it jumps out and I'm not sure why. cc @jyng

Reply:
We want to start merging some design language between Bridge and OS1. What other ways could we highlight that the star is the sponsor?

Reply:
> We want to start merging some design language between Bridge and OS1.

Oh, that's fair game I suppose. It had also somehow escaped my no

Good question re alternatives. I would like for this to mimic the login code design, which would put "Sponsor" in the head, and `~value` in the detail, but that would mean getting rid of the explanation in the detail... Maybe "Sponsor: `~value`" in the head then?

More broadly speaking, imo, this kind of small block next to text feels a lot like a category or warning or other kind of status indicator, and not at all like it's _the_ canonical descriptor for the data I'm looking at.